testpilot/frontend/static-src/app/main.js

```diff
@@ -40,6 +41,7 @@ app.extend({
   initialize() {
     app.webChannel = webChannel;
     app.me = new Me();
+    app.clipboard = new Clipboard('button');
```

Comment:
It doesn't add any new events (i.e. it uses bubbling correctly), and only affects buttons that have the appropriate `data-` attributes indicating what should be copied.